src/alert-manager/deploy/alert-templates/job-status-change/html.ejs

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html
+    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/xhtml"
+    style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
+
+<body>
+    Dear OpenPAI user:
+    </br>
+    </br>
+    Some of your jobs in OpenPAI cluster <b><%= cluster_id %></b> have changed status.
```

Comment:
Status of your job(s) has changed in OpenPAI cluster?

Reply:
> Status of your job(s) has changed in OpenPAI cluster?

Fixed